test/integration/client/client_test.go

```diff
@@ -802,3 +807,80 @@ func TestSelfLinkOnNamespace(t *testing.T) {
 
 	runSelfLinkTestOnNamespace(t, c, "default")
 }
+
+func TestApplyWithApplyConfiguration(t *testing.T) {
+	deployment := appsv1ac.Deployment("nginx-deployment-3", "default").
+		WithSpec(appsv1ac.DeploymentSpec().
+			WithSelector(metav1ac.LabelSelector().
+				WithMatchLabels(map[string]string{"app": "nginx"}),
+			).
+			WithTemplate(corev1ac.PodTemplateSpec().
+				WithLabels(map[string]string{"app": "nginx"}).
+				WithSpec(corev1ac.PodSpec().
+					WithContainers(
+						corev1ac.Container().
```

Comment:
If you combine this with the prior line, do we get rid of the unsightly indentation skip between line 835 and 836?

(Yes I feel bad commenting on something so petty)

Reply:
No problem! It works. we now have a clean cascade of closing params :)